ui/app/styles/core/alert-banner.scss

```diff
@@ -0,0 +1,7 @@
+// To match the updated design system, alert banner messages should have black text
```

Comment:
Personally I don't think this comment is needed, since it's clear what the styles are doing and one could look back at this PR for reasoning. But that's just a personal opinion/nit!

Reply:
Works for me!  I'll remove it.